src/reactviews/pages/QueryResult/slickgrid.tsx

```diff
@@ -180,7 +180,9 @@ const SlickGrid = forwardRef<SlickGridHandle, SlickGridProps>(
 
             let dataProvider = new HybridDataProvider(
                 collection,
-                (_startIndex, _count) => Promise.resolve([]),
+                (_startIndex, _count) => {
+                    return props.loadFunc(_startIndex, _count);
```

Comment:
Is this used during sorting?

Reply:
I see, no it's fine. Btw you might want to remove the `_` prefix from these parameters as they are being used now.